mmv1/third_party/terraform/framework_utils/framework_config.go.erb

```diff
@@ -625,14 +625,12 @@ func GetCredentials(ctx context.Context, data ProviderModel, initialCredentialsO
 
 	tflog.Info(ctx, "Authenticating using DefaultClient...")
 	tflog.Info(ctx, fmt.Sprintf("  -- Scopes: %s", clientScopes))
-	defaultTS, err := googleoauth.DefaultTokenSource(context.Background(), clientScopes...)
+	creds, err := transport.Creds(context.Background(), option.WithScopes(clientScopes...))
```

Comment:
Note: `DefaultTokenSource` calls `FindDefaultCredentials` and extracts the TokenSource from the credenetials (which we then use to reconstruct a credentials object...), Creds calls `FindDefaultCredentials` and returns the credentials.